server/handler/workflow_run.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handler
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/google/go-github/v63/github"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/palantir/policy-bot/policy/common"
+	"github.com/palantir/policy-bot/pull"
+	"github.com/pkg/errors"
+)
+
+type WorkflowRun struct {
```

Comment:
You'll also need to register this new handler with the `EventDispatcher` so it will actually get the events: https://github.com/palantir/policy-bot/blob/develop/server/server.go#L184-L192

Reply:
Oh yeah, thanks! [Here it is](https://github.com/palantir/policy-bot/compare/fd32004717210a737936d36f1039745bbeef1484..af8fa99485a8f8062b5ef9bdc0db05b6675489d2).

I suppose it mostly worked for us because you tend to get `check_run` events anyway...